src/app/static/src/app/store/baseline/baseline.ts

```diff
@@ -1,17 +1,19 @@
 import { Module } from 'vuex';
-import { actions } from './actions';
+import {actions, PJNZ} from './actions';
 import { mutations } from './mutations';
 import {RootState} from "../../main";
 
 export interface BaselineState {
     pjnzError: string
     country: string
-    complete: boolean
+    complete: boolean,
+    pjnzFilename: string
```

Comment:
I suspect we'll eventually restructure to have a 'FileUpload' type with error, filename properties, of which PJNZ will be one of several properties of this type in the state. But that doesn't need to go in now.

Reply:
Yeah this sounds sensible.